src/platform/packages/private/kbn-screenshotting-server/src/paths.ts

```diff
@@ -17,53 +17,58 @@ export interface PackageInfo {
   binaryChecksum: string;
   binaryRelativePath: string;
   isPreInstalled: boolean;
-  location: 'custom' | 'common';
+  location: 'custom' | 'CfT';
   revision: number;
```

Comment:
It seems we are not using revision anymore for anything. Can we remove it?

Reply:
You're right, I'd kept it around to somewhat denote the revision we are using, but I suppose that all information we can easily verify without having it specified.